vm/devices/net/netvsp/src/lib.rs

```diff
@@ -4115,7 +4116,12 @@ impl Coordinator {
                     .into_iter()
                     .filter(|&index| index < num_queues)
                     .collect::<Vec<_>>();
-                active_queues.len() as u16
+                if !active_queues.is_empty() {
+                    active_queues.len() as u16
+                } else {
+                    tracing::warn!("Invalid RSS indirection table");
```

Comment:
`tracelimit` here, too.

In general, `tracelimit` anywhere the event is guest controlled.